www/include/core/login/processLogin.php

```diff
@@ -44,7 +44,8 @@
         $_POST["autologin"] &&
         isset($generalOptions["enable_autologin"]) &&
         $generalOptions["enable_autologin"])
-    || (!isset($generalOptions['sso_enable']) || $generalOptions['sso_enable'] == 1)) {
+    || (!isset($generalOptions['sso_enable']) || $generalOptions['sso_enable'] == 1)
```

Comment:
you need to choose if you want the boolean operators to be at the end or the beginning of a line